### PR TITLE
Proof of Concept: Mascara

### PR DESCRIPTION
**Pull request by frankiebee**
This is the Working POC of mascara:

`background.js` is the service worker 
to spin up server: ` cd mascara && node server`
navigate to http://localhost:9001/popup/popup.html to create an account 

navigate to http://localhost:9002 to demo app hit sync tx.. (auto-faucet relies on XMLHttpRequest which is a forbidden api inside service workers fixed in PR#1306 depending on the state of that you may have to send some test eth to your account)

### CODE REVIEW

File: app/scripts/metamask-controller.js [hunk 1]
```diff
@@ -167,6 +167,7 @@ module.exports = class MetamaskController extends EventEmitter {
       rpcUrl: this.configManager.getCurrentRpcAddress(),
       // account mgmt
       getAccounts: (cb) => {
+        console.log('METAMASK CONTROLLER: getAccounts was called')
```
Comment (kumavis) [inline app/scripts/metamask-controller.js]: remove

Comment (kumavis) [inline app/scripts/metamask-controller.js]: remove

**Review by kumavis — COMMENTED** [review]
looks good - one log in metamask-controller

**Review by kumavis — APPROVED** [review]

**Comment by kumavis** [comment]
good work frankie!

**Comment by Zanibas** [comment]
I thought we were going to walk through this one together? @frankiebee lololol

**Comment by kumavis** [comment]
@Zanibas its pretty isolated, im getting ready to deploy a demo

**Pull request merged**